specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2021-09-09/examples/ImportImageByTag.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2021-06-01-preview",
```

Comment:
@AntonioVT please update these examples to use `2021-09-09` API version

Reply:
Fixed! I missed updating the api-version for the files I missed.